src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsMapper.cs

```diff
@@ -0,0 +1,100 @@
+// <copyright company="Dynatrace LLC">
+// Copyright 2021 Dynatrace LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using OpenTelemetry.Metrics.Export;
+using DynatraceMetric = Dynatrace.MetricUtils.Metric;
+using DynatraceMetricFactory = Dynatrace.MetricUtils.MetricsFactory;
+using System.Text;
+
+internal static class DynatraceMetricsMapper
+{
+
+	/// <summary>
+	/// Combines metric namespace and key into a single key for use in <see cref="Dynatrace.MetricUtils.Metric">.
+	/// </summary>
+	/// <returns>A metric key in the form {MetricNamespace}.{MetricKey}</returns>
+	private static string CreateMetricKey(Metric metric)
+	{
+		var keyBuilder = new StringBuilder();
+		if (!string.IsNullOrEmpty(metric.MetricNamespace))
+		{
+			keyBuilder.Append($"{metric.MetricNamespace}.");
+		}
+		keyBuilder.Append(metric.MetricName);
+		return keyBuilder.ToString();
+	}
+
+	internal static IEnumerable<DynatraceMetric> ToDynatraceMetric(Metric metric)
+	{
+		var metricName = CreateMetricKey(metric);
+		foreach (var metricData in metric.Data)
+		{
+			var timestamp = metricData.Timestamp;
+			var dimensions = metricData.Labels;
+
+			switch (metric.AggregationType)
+			{
+				case AggregationType.DoubleSum:
+					{
+						var sum = metricData as DoubleSumData;
+						var dynatraceMetric = DynatraceMetricFactory.CreateDoubleCounterDelta(metricName: metricName,
```

Comment:
This might be the tiniest of nits but could we indent the metric name as well? It somehow bothers me more than it should :grimacing: 
```suggestion
						var dynatraceMetric = DynatraceMetricFactory.CreateDoubleCounterDelta(
																							 metricName: metricName,
```

Reply:
I agree, it's quite annoying. :sweat_smile: Seems to be a tabs & spaces situation. I will change occurrences of this, but I suspect it might appear again for people in the future depending on their IDE settings.

Reply:
Fixed in [844fd4e](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/24/commits/844fd4e7b3d76bd088e669481ab057509e5b4ad0)

Reply:
Actually fixed as of [ee75500](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/24/commits/ee755002d90d4287405817b2b62085dce960fba4). Sorry for the misunderstanding. :)